lib/helpers/umzug-helper.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var args = require("yargs").argv;
+var path = require("path");
+var _    = require("lodash");
+
+module.exports = {
+  getStorageMethod: function () {
+    return args.storage || "sequelize";
```

Comment:
I'm fine with using `fs`. tbh I kinda regret introducing gulp to the tests. Anyways, this is how you can use the readFile pattern:

```
 gulp
          .src("dir")
          .pipe(helpers.readFile("sequelize-meta.json"))
          .pipe(helpers.expect(function(stdout) {
              expect(stdout).to.contain("up: function(migration, DataTypes, done) {");
          })
          .pipe(helpers.teardown(done));
```